docs/graphql/manual/deployment/graphql-engine-flags/reference.rst

```diff
@@ -196,6 +196,10 @@ For ``serve`` sub-command these are the flags and ENV variables available:
      - Set the logging level. Default: ``info``. Options: ``debug``, ``info``,
        ``warn``, ``error``.
 
+   * - ``--enable-compression``
+     - ``HASURA_GRAPHQL_ENABLE_COMPRESSION``
+     - Enable brotli/gzip compression for responses from ``/v1/query`` and ``/v1/graphql`` endpoints.
```

Comment:
Is there a default?

Reply:
I meant, is it enabled by default? Other `ENABLE_*` settings already state a default, usually `false` but at least it makes it less ambiguous.